srcpkgs/libresprite/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'libresprite'
+pkgname=libresprite
+version=1.0
+revision=1
+archs="x86_64"
+create_wrksrc=yes
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE:STRING=Release -DWITH_DESKTOP_INTEGRATION=ON"
+hostmakedepends="git zlib-devel libpng-devel freetype-devel giflib-devel gtest-devel libjpeg-turbo-devel libcurl-devel SDL2-devel SDL2_image-devel lua-devel pixman-devel tinyxml-devel nodejs-devel"
+short_desc="Animated sprite editor & pixel art tool"
+maintainer="Mohammad Ebrahimi <moh.ebrahimi2000@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://libresprite.github.io"
+
+do_fetch() {
+	git clone --recursive https://github.com/LibreSprite/LibreSprite.git $wrksrc
```

Comment:
We want fast rebuild without cloning every time. List submodules like in ppsspp does or ask upstream for combined tarball.